requery/src/main/java/io/requery/sql/RawTupleNamedQuery.java

```diff
@@ -0,0 +1,166 @@
+package io.requery.sql;
+
+import io.requery.query.BaseResult;
+import io.requery.query.Expression;
+import io.requery.query.MutableTuple;
+import io.requery.query.NamedExpression;
+import io.requery.query.Result;
+import io.requery.query.Tuple;
+import io.requery.query.element.QueryType;
+import io.requery.util.CloseableIterator;
+import io.requery.util.function.Supplier;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Raw Query with Named parameters
+ *
+ * @author debop (Sunghyouk Bae)
+ * @since 18. 12. 8
+ */
+public class RawTupleNamedQuery extends PreparedQueryOperation implements Supplier<Result<Tuple>> {
```

Comment:
similarly this duplicates much RawTupleQuery so probably should extend it

Reply:
If I change RawTupleQuery to overridable, I will extend it